cds_migrator_kit/records/log.py

```diff
@@ -68,16 +68,13 @@ def get_stat_by_recid(recid, stats_json):
 
     def render_stats(self):
         """Load stats from file as json."""
-        # try:
-        logger.warning(self.LOG_FILEPATH, '----')
-        with open(self.LOG_FILEPATH, "r") as f:
-            all_stats = json.load(f)
-        # except IOError:
-        #     all_stats = []
-        # except ValueError:
-        #     all_stats = []
-
-            return all_stats
+        logger.warning(self.LOG_FILEPATH)
+        try:
+            with open(self.LOG_FILEPATH, "r") as f:
+                all_stats = json.load(f)
+                return all_stats
+        except Exception as e:
```

Comment:
Just a minor thing, I don't think this try..catch is needed at all anymore. If an exception is raised there's no need to re-raise it.